src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Long> ticketIds;
+    private Long userId;
```

Comment:
private Long userId;

this is redundant here;

Reply:
I don't think so. Let's see what mentor will say.